client/components/README.md

```diff
@@ -1,8 +1,10 @@
 Components
 ==========
 
-_Read the [documentation on components](https://github.com/Automattic/wp-calypso/blob/master/docs/components.md) for more in-depth information._
+Components are React components used for composing the UI of Calypso.
 
-This place harbors shared React components used for composing the UI of Calypso. Components come with their own styles defined [according to our guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md), and manually loaded from the [styles assets folder](https://github.com/Automattic/wp-calypso/blob/master/assets/stylesheets/_components.scss). Structuring the user interface with these building blocks has several benefits — like allowing to quickly construct a view that is visually consistent with the rest of Calypso, and easier to iterate on.
+They come with their own styles defined [according to our guidelines](../../docs/coding-guidelines/css.md), and manually loaded from the [styles assets folder](../../assets/stylesheets/_components.scss). Structuring the user interface with these building blocks has several benefits — like allowing to quickly construct a view that is visually consistent with the rest of Calypso, and easier to iterate on.
```

Comment:
Maybe choose a different word than `block` in `building blocks`?

Reply:
Haha. I had the same thought when refining this section, but couldn't come up with a better term.